pages/Threat_Modeling_Process.md

```diff
@@ -42,46 +44,59 @@ This document describes a structured approach to application threat modeling tha
 
 Threat modeling looks at a system from a potential attacker's perspective, as opposed to a defender's viewpoint. Making threat modeling a core component of your [SDLC](https://en.wikipedia.org/wiki/Systems_development_life_cycle) can help increase product security.
 
-The threat modeling process can be decomposed into three high level steps. Each step is documented as it is carried out. The resulting document is the threat model for the application.
+The threat modeling process can be decomposed into four high level steps. Frequently, each step is documented as it is carried out. The resulting document is the threat model for the application. Experts distinguish between the work of doing threat modeling and the work to create threat model documents.
 
-### Step 1: Decompose the Application
+### Step 1: Scope your work
 
-The first step in the threat modeling process is concerned with gaining an understanding of the application and how it interacts with external entities. This involves:
+The first step in the threat modeling process is concerned with gaining an understanding of what you're working on. This can  involve:
```

Comment:
```suggestion
The first step in the threat modeling process is concerned with gaining an understanding of what you're working on. This can involve:
```